src/notifications/ManagerLinux.cpp

```diff
@@ -187,11 +187,14 @@ NotificationsManager::systemPostNotification(const QString &room_id,
     // The list of actions has always the action name and then a localized version of that
     // action. Currently we just use an empty string for that.
     // TODO(Nico): Look into what to actually put there.
-    argumentList << (QStringList(QStringLiteral("default"))
-                     << QLatin1String("") << QStringLiteral("inline-reply")
-                     << QLatin1String("")); // actions
-    argumentList << hints;                  // hints
-    argumentList << (int)-1;                // timeout in ms
+    QStringList actions;
+    actions << QStringList(QStringLiteral("default")) << QLatin1String("");
+    if (!room_id.isEmpty()) {
```

Comment:
I think you need a similar logic also on macOS

Reply:
@redsky17: could you help out here? :)

Reply:
Yeah, I can test it out.

Reply:
Okay, so I don't know exactly what I'm looking for and when.  I thought I'd be able to test it out just by keeping nheko closed for a bit to let messages roll in... but I'm not seeing any differences in behavior.  Sooo... what should I be looking for?

Reply:
Thanks for the overview.  On second review, it looks like notifications might be broken right now on macOS.  What I thought was existing functionality was actually just old notifications I hadn't cleared out.  I'll have to debug it more later

Reply:
Okay, I can confirm that notifications *aren't* broken.  I think one of my other devices was consuming the messages so I wasn't getting any notifications on my macbook.  I unfortunately left nheko open so I didn't get the 'digest' notification.  I'm going to stop nheko for a while and see what happens when I start it back up.

Reply:
I'm not getting any notifications on start now after being away for a while, so it seems like the one function call isn't working as expected.  It's hard to debug these issues, because they never rely in errors on the client side, but I'm thinking you probably have to call another overload of the function that doesn't take a 'replyActions' parameter.

Reply:
A random thought: Maybe is it because we use the same identifier for two different categories (one with replies and one without)?